filebeat/spooler/spooler.go

```diff
@@ -33,18 +37,17 @@ type spoolerConfig struct {
 // started by calling Start before it can be used.
 func New(
 	config *cfg.Config,
-	publisher chan<- []*input.Event,
+	out Output,
```

Comment:
I'd rather do not. Output is the interface any component must implement. It's part of spooler's interface as reusable component.